service/routes.py

```diff
@@ -122,8 +130,8 @@ def index():
 ######################################################################
 #  PATH: /wishlists/{id}
 ######################################################################
-@api.route('/wishlists/<wishlist_id>')
-@api.param('wishlist_id', 'The Wishlist identifier')
+# @api.route('/wishlists/<wishlist_id>')
+# @api.param('wishlist_id', 'The Wishlist identifier')
```

Comment:
Commented these out because some tests started to fail. It sends the requests to these new methods now, and since their empty - nothing works.